src/main/java/core/basesyntax/shop/db/FruitShopStorage.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.shop.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitShopStorage {
+    private static final Map<String, Integer> FRUIT_SHOP_MAP = new HashMap<>();
+
+    public static Map<String, Integer> getFruitShopMap() {
+        return FRUIT_SHOP_MAP;
+    }
```

Comment:
Seems u need to create StorageDao or create this method in the another class.

Reply:
It's a getter for the map. without it i should make this map public

Reply:
Yes you should.
```suggestion

    public static final Map<String, Integer> storage = new HashMap<>();
    }
```

Reply:
ok let's ask mentor, because i see the breaking of incapsulation principle here